### PR TITLE
Shell1 Edits from Jared (1)

### PR DESCRIPTION
**Pull request by jaredawebb**
There were several small errors that needed to be corrected. I also simplified language and improved flow. However, the first and second half aren't as unified as I would like.

One exercise remains to be written (See line 250.  Should take 5 minutes).
At the very beginnning of the lab there was some text that seemed important but was missing. Joey should look at it and fix it. I tried to make apparent where it was in the edited document.

### CODE REVIEW

**Comment by shanemcq18** [comment]
See Issue #1284. @jaredawebb, which problem still needs to be written?

File: Labs/Vol3A/UnixShell1/shell1.tex [hunk 1]
```diff
@@ -6,24 +6,32 @@
...
+The majority of servers are Linux based, so having a knowledge of Unix shell commands allows us to interact with these servers. 
 
 As you get into Unix, you will find it is easy to learn but difficult to master.
 We will build a foundation of simple file system management and a basic introduction to the Vim text editor.  
 We will address some of the basics in detail and also include lists of commands that interested learners are encouraged to research further.
 
-\begin{info} sary update so you will get the same functionality as these other systems.
+\begin{info} 
+WHAT IS THIS SAYING????  Ask Joey.
+sary update so you will get the same functionality as these other systems.
```
Comment (shanemcq18) [inline Labs/Vol3A/UnixShell1/shell1.tex]: @jadams1988 this line can just be deleted, right?

File: Labs/Vol3A/UnixShell1/shell1.tex [hunk 2]
```diff
@@ -246,7 +286,8 @@ \subsection*{Pipes and Redirects}
 %$
 
 In the previous example, we pipe the contents of \li{assignments.txt} to \li{wc -l} using \li{cat}.
-When working with files specifically, it is better to use \emph{redirects}.
+When working with files specifically, you can also use \emph{redirects}.
+The $\li{<}$ operator gives a file to a Terminal command.
```
Comment (shanemcq18) [inline Labs/Vol3A/UnixShell1/shell1.tex]: This line keeps the file from compiling. The dollar signs need to be removed.

File: Labs/Vol3A/UnixShell1/shell1.tex (hunk 1 above)
Comment (jaredawebb) [inline Labs/Vol3A/UnixShell1/shell1.tex]: Maybe?  I wasn't sure what you were trying to say.

File: Labs/Vol3A/UnixShell1/shell1.tex (hunk 2 above)
Comment (jaredawebb) [inline Labs/Vol3A/UnixShell1/shell1.tex]: I think I fixed it?

Comment (jaredawebb) [inline Labs/Vol3A/UnixShell1/shell1.tex]: Also, I mentioned where the new exercise should be written.  If anyone else thinks its a good idea.

**Comment by shanemcq18** [comment]
Closing in favor of #1285.

**Pull request closed without merging**